query-engine/connector-test-kit-rs/query-engine-tests/tests/new/regressions/prisma_15467.rs

```diff
@@ -80,13 +80,15 @@ db.Standing.updateMany({{
         let expected_query = query.trim();
         assert!(
             last_log_line.contains(expected_query),
+            "{}",
             r#"{last_log_line} should have contained {expected_query}"#,
         );
 
         // Piggybacking assertion reproducing https://github.com/prisma/prisma/issues/14378
         let expected_duration_field = "duration_ms";
         assert!(
             last_log_line.contains(expected_duration_field),
+            "{}",
             r#"{last_log_line} should have contained {expected_duration_field}"#
```

Comment:
I'm pretty sure this breaks the assertion message, the template string won't be formatted if it's in format argument position.

Reply:
I see this change was introduced by Jan to fix the CI on Friday. I'll take a look at how it looks like in test's output

Reply:
Will be fixed by https://github.com/prisma/prisma-engines/pull/3643

Reply:
Tested `cargo test --package query-engine-tests --test query_engine_tests -- new::regressions::prisma_15467::mongodb::update_many_log_output --exact --nocapture` locally, is it ok now?